### ISSUE TITLE
Current Progress

### ISSUE THREAD

**Issue by Dankhaze420g** [Electrical-Age/ElectricalAge#933]
this is an very amazing mod i'm just curious about how far along the port is? it be great to have this on 1.2.2 since most mods are on that version now

**Comment by jrddunbr** [comment]
Lack of very active developers means it's going at a snail's pace. :/

**Comment by Dankhaze420g** [comment]
yeah ive been checking it out about once a year since the last release it sucks hopefully something soon
